xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -518,6 +518,13 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
           m_formatname = isDvhe ? "amc-dvhe" : "amc-dvh1";
         }
       }
+      
+      else if (m_hints.codec_tag == MKBETAG('d', 'v', 'v', 'C') ||
```

Comment:
It should not bypass the display capabilities in the condition above.
This should go into `isDvh1` instead.

Reply:
Ok, but we still need to check for MKBETAG dvvC and dvcC. So there are a few possible ways of doing so:

1. Create two more boolean variables: isDvvC and isDvcC, check for the MKBETAG there and add to the If statement along with isDvhe and isDvh1. 

Or

2. Add both MKBETAG to the isDvh1, change the variable name for something that accomodates everything like isDovi.

I'm open to suggestions and I'll update this PR.

Reply:
It has to be option 2 just because `m_formatname` only has `dvhe` and `dvh1`.  
IMO the variable can keep the `isDvh1` name.

Reply:
Made the changes suggested by you and Jenkins. 
My VSCode is set to indent with 4 spaces every new line and I didn't catch it before.

Compiled it again and tested with the Nvidia Shield. It still works. 

Variable is still named dvh1 but I think isDovi or something more generic would suit it better.
I'll replace the current one on my release page.